conventional_precommit_linter/hook.py

```diff
@@ -171,9 +174,15 @@ def print_report(commit_type: str, commit_scope: Optional[str], commit_summary:
     rule_messages.append(
         f"{_get_icon_for_rule(rules_output_status['error_scope_format'])} {_color_blue('(<optional-scope>)')} if used, must be enclosed in parentheses"
     )
-    rule_messages.append(
-        f"{_get_icon_for_rule(rules_output_status['error_scope_capitalization'])} {_color_blue('(<optional-scope>)')} if used, must be written in lower case without whitespace"
-    )
+
+    if args.allow_uppercase_in_optional_scope:
+        rule_messages.append(
+            f"{_get_icon_for_rule(rules_output_status['error_scope_capitalization'])} {_color_red('overridden with --allow-uppercase-in-optional-scope')} {_color_blue('(<optional-scope>)')} if used, must be written in lower case without whitespace."
```

Comment:
This output message is confusing.

1) `overridden with --allow-uppercase-in-optional-scope` part is not necessary and breaks the output format style
2) if the condition changed, then also output message should change

I suggest something like `if used, must be written without whitespace.` for that case.